operator_csv_libs/imagerepo.py

```diff
@@ -200,6 +202,72 @@ def _get_quay_repo(self):
         r = self.image.get_image_repo().replace('quay.io/','')
         return '/'.join([r, self.image.get_image_name()])
 
+class DockerRepo:
+    
+    def __init__(self,image, docker_user=None, docker_key=None):
+        self.image = image
+        self.org = image.get_image().split('/')[1]
+        self.repo = image.get_image_name()
+        self.tag = image.get_tag()
+
+        if docker_user:
+            self.docker_user = docker_user
+        elif 'DOCKER_USER' in os.environ:
+            # Go to environment variables if we don't have them as class variable either
+            self._docker_user = os.getenv('DOCKER_USER')
```

Comment:
So, it looks like we're looking for and storing the user info, but we don't ever use it?
I think either we remove that, or if we accept them, we'll use them in the token section if we have them.
This would allow us to get digest for private images if access info is provided.